lib/std/zig/system.zig

```diff
@@ -645,7 +646,7 @@ pub const NativeTargetInfo = struct {
                     {
                         var dyn_off = elfInt(is_64, need_bswap, ph32.p_offset, ph64.p_offset);
                         const p_filesz = elfInt(is_64, need_bswap, ph32.p_filesz, ph64.p_filesz);
-                        const dyn_size: u64 = if (is_64) @sizeOf(elf.Elf64_Dyn) else @sizeOf(elf.Elf32_Dyn);
+                        const dyn_size: usize = if (is_64) @sizeOf(elf.Elf64_Dyn) else @sizeOf(elf.Elf32_Dyn);
```

Comment:
Does peer-type resolution not work here? (i.e. omit the type?)

Reply:
`is_64` is runtime known, so it hits  #137

Reply:
oh interesting. I thought `@sizeOf` would return a comptime-known `usize` rather than a `comptime_int`